ow_lander/scripts/camera_capture_action_client.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import actionlib
+import argparse
+import ow_lander.msg
+from guarded_move_action_client import print_arguments
+
+
+def camera_capture_client():
+  parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+  parser.add_argument(
+    'exposure', type=float, help='Camera exposure in seconds', nargs='?', default=-1, const=0)
+  args = parser.parse_args()
+  print_arguments(args)
+
+  client = actionlib.SimpleActionClient('CameraCapture', ow_lander.msg.CameraCaptureAction)
+  client.wait_for_server()
+
+  goal = ow_lander.msg.CameraCaptureGoal(exposure=args.exposure)
```

Comment:
If we keep exposure, would it make sense to define a valid range?  I found that values over 1 seemed unusable and indistinguishable.  If there's a valid range, it should be checked.

Reply:
Valid range is anything > 0. If you were taking pictures at night, you might want a very high exposure. (Of course, a camera on a real mission would probably have a range set by hardware and software constraints.)